mcs/tools/wasm-tuner/tuner.cs

```diff
@@ -76,12 +89,94 @@ public class WasmTuner
 		}
 	}
 
+	void Usage () {
+		Console.WriteLine ("Usage: tuner.exe <arguments>");
+		Console.WriteLine ("Arguments:");
+		Console.WriteLine ("--gen-icall-table icall-table.json <assemblies>.");
+		Console.WriteLine ("--gen-pinvoke-table <list of native library names separated by commas> <assemblies>.");
+	}
```

Comment:
The `Usage:` seems to suggest that one can specify both `--gen-icall-table` and `--gen-pinvoke-table` arguments but below it it seems to to only process `--gen-icall-table` or `--gen-pinvoke-table`.  Is that correct?

Reply:
Currently yes, later we want both of them so it only needs to run once.